posthog/api/app_metrics.py

```diff
@@ -74,6 +75,50 @@ def error_details(self, request: request.Request, *args: Any, **kwargs: Any) ->
         error_details = AppMetricsErrorDetailsQuery(self.team, plugin_config.pk, filter).run()
         return response.Response({"result": error_details})
 
+    def get_batch_export_runs_app_metrics_queryset(self, batch_export_id: str):
+        """Use the Django ORM to fetch app metrics for batch export runs.
+
+        Attempts to (roughly) match the following (much more readable) query:
+        ```
+        select
+            date_trunc('day', last_updated_at) as dates,
```

Comment:
maybe add a comment about why we're using `last_updated_at` and what it means

Reply:
Sure thing :+1: